src/doc/tutorials/PythonExpressions.rst

```diff
@@ -0,0 +1,202 @@
+.. _Tutorials_Python_Expressions:
+
+Python Expressions
+==================
+
+.. toctree::
+       :maxdepth: 2
+
+This tutorial describes using Python Expressions.
+
+Python expressions overview
+---------------------------
+
+VisIt_ contains the ability to create expressions using the Python scripting language.
+Python expressions run in the compute engine and run in parallel when VisIt_ is run in parallel.
+Python expressions operate on vtkDataSets, which provide access to variables as well as the mesh coordinates and topology.
+Python expressions also have access to VisIt_'s metadata as well as access to MPI when running in parallel.
```

Comment:
```suggestion
VisIt_'s Python Expression system also provides access to methods in VisIt_'s CLI module (e.g. ``visitmodule`` as well as Python MPI methods when running in parallel.
```

Reply:
Don't we also provide `numpy` in this env. too?

Reply:
I'm not sure about access to visitmodule. visitmodule is used to drive VisIt and I don't believe using it would work. Numpy is definitely available.

Reply:
Hmmm...do you reall mean `avtDatabaseMetadata` and friends in the `AVT` library? When you say *...like avtMetaData...*, I think you mean like anything available through **CLI** functions.

Reply:
I really mean things like avtMetaData. I don't think using the functions from visitmodule would actually work, at least it wouldn't do what you want. For example, if you did an `AddPlot` or `SetPlotOptions`, I don't think things are wired up to have that impact the current VisIt instance. I suppose you could launch a new instance of VisIt and do stuff with it, but I'm not sure we want people doing that.

Reply:
Ah, ok. I totally misunderstood then. I thought all the **CLI** functions are indeed available too...even if some wouldn't necessarily make sense to use.